google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -682,6 +686,82 @@ def copy destination_table, create: nil, write: nil, dryrun: nil,
         end
 
         ##
+        # Copies the data from the table to another table using a synchronous
+        # method that blocks for a response. In this method, a {CopyJob} is
+        # created and the service is polled until the job is done. Timeouts and
+        # transient errors are generally handled as needed to complete the job.
+        # See also #{#copy_job}.
+        #
+        # @param [Table, String] destination_table The destination for the
+        #   copied data. This can also be a string identifier as specified by
+        #   the [Query
+        #   Reference](https://cloud.google.com/bigquery/query-reference#from):
+        #   `project_name:datasetId.tableId`. This is useful for referencing
+        #   tables in other projects and datasets.
+        # @param [String] create Specifies whether the job is allowed to create
+        #   new tables. The default value is `needed`.
+        #
+        #   The following values are supported:
+        #
+        #   * `needed` - Create the table if it does not exist.
+        #   * `never` - The table must already exist. A 'notFound' error is
+        #     raised if the table does not exist.
+        # @param [String] write Specifies how to handle data already present in
+        #   the destination table. The default value is `empty`.
+        #
+        #   The following values are supported:
+        #
+        #   * `truncate` - BigQuery overwrites the table data.
+        #   * `append` - BigQuery appends the data to the table.
+        #   * `empty` - An error will be returned if the destination table
+        #     already contains data.
+        #
+        # @return [Google::Cloud::Bigquery::CopyJob]
+        #
+        # @example
+        #   require "google/cloud/bigquery"
+        #
+        #   bigquery = Google::Cloud::Bigquery.new
+        #   dataset = bigquery.dataset "my_dataset"
+        #   table = dataset.table "my_table"
+        #   destination_table = dataset.table "my_destination_table"
+        #
+        #   copy_job = table.copy destination_table
+        #
+        # @example Passing a string identifier for the destination table:
+        #   require "google/cloud/bigquery"
+        #
+        #   bigquery = Google::Cloud::Bigquery.new
+        #   dataset = bigquery.dataset "my_dataset"
+        #   table = dataset.table "my_table"
+        #
+        #   copy_job = table.copy "other-project:other_dataset.other_table"
+        #
+        # @!group Data
+        #
+        def copy destination_table, create: nil, write: nil
+          ensure_service!
```

Comment:
Shouldn't this just call `#copy_job` directly? So all this logic isn't duplicated?